src/app/experts/self-tags/self-tags.component.ts

```diff
@@ -1,25 +1,28 @@
 import { Component, OnInit } from '@angular/core';
-import { NgForm } from '@angular/forms';
-import { HttpClient } from '@angular/common/http';
-
+import {Tags} from '../../models/Tags';
+import { TagService } from 'src/app/services/tags.service';
 
 
 
```

Comment:
indenting in this file needs to be fixed

Reply:
used the alt shift f command to make changes.